pkg/registry/apis/secret/encryption/cipher/provider/decipher_aescfb.go

```diff
@@ -0,0 +1,41 @@
+package provider
+
+import (
+	"context"
+	"crypto/aes"
+	cpr "crypto/cipher"
+
+	"github.com/grafana/grafana/pkg/registry/apis/secret/encryption/cipher"
+)
+
+var _ cipher.Decrypter = aesCfbDecipher{}
+
+type aesCfbDecipher struct{}
+
+func (aesCfbDecipher) Decrypt(_ context.Context, payload []byte, secret string) ([]byte, error) {
```

Comment:
Question, for new secrets manager, do we need AES-CFB if we are starting from scratch? Do you mean this for a potential migration path from old secrets?

Reply:
It really depends on whether we want to support the idea of "legacy format", which we currently do. That would entail decrypting CFB, unfortunately.